types/block.go

```diff
@@ -29,8 +29,9 @@ func MakeBlock(height int64, txs []Tx, commit *Commit) *Block {
 	block := &Block{
 		Header: &Header{
 			Height: height,
-			Time:   time.Now(),
+			Time:   time.Now().Round(0), // Strip monotonic.
```

Comment:
we use .UTC everywhere else ? Should just have `types.Now()`

Reply:
It should always be Round(0).UTC().